packages/react/src/components/textInput/TextInput.tsx

```diff
@@ -72,14 +68,6 @@ export type TextInputProps = MergeAndOverrideProps<
      * Short hint displayed in the input before the user enters a value
      */
     placeholder?: string;
-    /**
-     * If `true`, prevents the user from changing the value of the field (not from interacting with the field)
-     */
-    readOnly?: boolean;
-    /**
-     * If `true`, the label is displayed as required and the `input` element will be required
-     */
```

Comment:
The prop is native and therefore useless to list here, but setting the value `true` affects also the visuals (adds the "*"), so that should be mentioned somewhere. It is not in docs at the moment.

Perhaps it should be?

Reply:
I'll add

Reply:
Hmm, on site there's this "To decrease a cognitive load of the user, it is recommended to mark all required fields. Remember to provide explanation of the required field indicator at the top of your form." ping @NikoHelle

Reply:
Yes, but I think we should somewhere clearly say that if required=true there will be an asterix added after the label.

This part of JSDocs is nowhere after deletion "the label is displayed as required".  Should it be moved to docs?

Reply:
Looks like this now on the doc-site 
![image](https://github.com/user-attachments/assets/680a49ed-1df5-4ce0-8856-b1bf2f73eaaa)

Reply:
ping @laurakarhu is this ☝️ ok?